tox.ini

```diff
@@ -4,31 +4,15 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = docs, py{27,34}pep8, py{26,27,33,34,py}, py{26,27,33,34,py}-defusedxml, py{27,34}-coverage
+envlist = py{27,38}
```

Comment:
Remove old and redundant environments. Get us to what's in production (2.7) and where we need to go (3.8).